examples/README.md

```diff
@@ -0,0 +1,85 @@
+# krun example
+
+This directory contains a simple experiment using krun.
+This is a good starting point for setting up your own krun configuration.
+
+The example here contains two benchmark programs (*nbody* and *dummy*),
+executed on two VMs (*cPython* and a standard *JVM* such as HotSpot).
+Each benchmark is run for 5 iterations on the same VM, then the VM is
+restarted and the benchmark is re-run for another 5 iterations.
+We say that the benchmarks are *executed* twice, with 5 *iterations*.
+
+This configuration can be found in the file `examples/example.krun`.
+
+## Step 0: prepare the benchmarking machine
+
+To run this example experiment, you need `sudo` rights to the machine you are
```

Comment:
I think it's OK in the documentation to just mention `sudo` since that's the dominant way of doing such things. Other platforms which don't use `sudo` will work out what to replace it with.